src/natcap/invest/ui_server.py

```diff
@@ -6,15 +6,20 @@
 import json
 import logging
 import pprint
+import sys
 import textwrap
 
 from flask import Flask
 from flask import request
 import natcap.invest.cli
 import natcap.invest.datastack
+from natcap.invest.utils import LOG_FMT
 
-logging.basicConfig(level=logging.DEBUG)
 LOGGER = logging.getLogger(__name__)
+handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter(LOG_FMT)
+handler.setFormatter(formatter)
+LOGGER.addHandler(handler)
```

Comment:
This makes sense to me for the context of the `ui_server` application, but I did want to check about the intent ...  `logging.basicConfig` sets up the root logger (if the root logger doesn't already have a handler), and this change will set up a logger for `natcap.invest.ui_server`, which means that the logger as configured will only capture logging from the application, not from any models, utilities, or anything else like that.  Is that the intent?

Reply:
Thanks, @phargogh . No that is not the intent. The intent is definitely to capture all the messages from all the modules and stream them to stdout. I'll give it another go.